lib/worker.js

```diff
@@ -296,6 +293,8 @@ class AssetComputeWorker {
 
         // post-processing // TODO: Refactor this to share partas with post-processing in `processRendition` func
         for (let rendition of this.renditions) {
+            // start post-processing timer
+            const postProcessingTimer = this.timers.currentPostProcessing = new Timer();
```

Comment:
the only reason you are doing `postProcessingTimer` inside for(..) is because we care for the `postProcessingTimer.durationSec()` in logs correct? otherwise Metrics wise.. this can be done outside for..loop.
```
const postProcessingTimer = this.timers.currentPostProcessing = new Timer();

this.timers.postProcessing += postProcessingTimer.duration();
```

Reply:
the reason I did this in the loop was for two reasons:
1. I wanted to keep the post-processing functions the same in `worker` and `batchWorker` since we plan to extract this function out and use it in both cases
2. Yes the timestamp is needed for logging purposes, not metrics as we don't track metrics one by one for each rendition in batchWorker